test/e2e/overlays.rendering.test.ts

```diff
@@ -258,3 +265,24 @@ describe('Overlay navigation', () => {
     });
   });
 });
+
+describe('Overlay style', () => {
+  const bpmnDiagramName = 'overlays.start.flow.task.gateway';
+  const snapshotPath = 'with.custom.style';
+
+  it.each(['font', 'fill', 'stroke'])(`add overlay with custom %s`, async (style: string) => {
+    await pageTester.loadBPMNDiagramInRefreshedPage(bpmnDiagramName);
+
+    await addStylingOverlay(['StartEvent_1', 'Activity_1', 'Gateway_1', 'Flow_1'], style);
+
+    const image = await page.screenshot({ fullPage: true });
+    const config = imageSnapshotConfigurator.getConfig(bpmnDiagramName);
+    expect(image).toMatchImageSnapshot({
+      ...config,
+      failureThreshold: 0.02, // 1.926162542254506%
```

Comment:
⚠️ same high threshold value (more than 1%) for all OS and browser 😒 This could prevent detecting small changes

Reply:
We have decided with @csouchet to merge as it is now and manage it right after in a dedicated PR.